src/login/org.freedesktop.login1.policy

```diff
@@ -2,16 +2,7 @@
 <!DOCTYPE policyconfig PUBLIC "-//freedesktop//DTD PolicyKit Policy Configuration 1.0//EN"
         "http://www.freedesktop.org/standards/PolicyKit/1/policyconfig.dtd">
 
-<!--
-  SPDX-License-Identifier: LGPL-2.1+
-
-  This file is part of systemd.
-
-  systemd is free software; you can redistribute it and/or modify it
-  under the terms of the GNU Lesser General Public License as published by
-  the Free Software Foundation; either version 2.1 of the License, or
-  (at your option) any later version.
--->
+<!-- SPDX-License-Identifier: LGPL-2.1+ -->
```

Comment:
We have not done that yet. If do like this, all policy file should be updated.

Reply:
So far we kept verbose copyright headers in installed files. I think it would be reasonable to drop them, but I think we should do that globally then.